client/api/request.js

```diff
@@ -21,7 +27,7 @@ client.interceptors.request.use(config => {
 });
 
 client.interceptors.response.use(res => res, err => {
-  if (err.response.status === 401) {
+  if (err.response && err.response.status === 401) {
```

Comment:
🎖 🤡

Reply:
Btw, anybody for enabling optional chaining (https://github.com/tc39/proposal-optional-chaining)?

Reply:
Here is what `ag` thinks of it :neckbeard: 
```
$ ag '(\w+)\s&&\s\1\.' -C1

server/router.js
32-  const { body } = req;
33:  if (body && body.email) body.email = body.email.toLowerCase();
34-  next();

client/components/content-elements/tce-table/edit/index.vue
48-    anchor.position,
49:    (adjacent && adjacent.position) || limit(anchor, direction)
50-  );

client/components/course/TreeView/index.vue
100-  const desc = find(structure, { type });
101:  return desc && desc.color;
102-}

client/components/editor/Toolbar.vue
11-    <element-toolbar
12:      v-if="element && element.parent"
13-      :key="`${element.parent._cid}-${element.id}`"

client/components/editor/structure/Assessments.vue
67-      const question = assessment.data.question;
68:      const hasQuestion = question && question.length > 0;
69-      if (this.isSelected(assessment) && !hasQuestion) {

client/components/editor/structure/Exams/AssessmentGroup.vue
103-      const question = assessment.data.question;
104:      const hasQuestion = question && question.length > 0;
105-      if (this.isSelected(assessment) && !hasQuestion) {

client/store/helpers/collection.js
49-        const previous = this.context.state.items[model._cid];
50:        if (previous && previous._version === model._version) model._synced
 = true;
51-        this.commit('save', model);
```

Reply:
Because we use get 😉

Reply:
You lodash people 👾

Reply:
> Btw, anybody for enabling optional chaining ([tc39/proposal-optional-chaining](https://github.com/tc39/proposal-optional-chaining))?

Why not? I'm up for it. It's in stage 2 which means it's arriving in the spec soon.